brands/BOM/src/index.js

```diff
@@ -1,7 +1,9 @@
-// @codegen
+import codegen from 'babel-plugin-codegen/macro';
```

Comment:
Is this necessary?

Reply:
Very yes

Reply:
More words needed.

Reply:
@mitchellhamilton can you remember exactly why this is?

Reply:
It's because we can't customise the Admin UI's webpack/babel config so we need to use the babel macro for codegen instead of the babel plugin